domain/src/main/java/org/fao/geonet/repository/MetadataRepositoryImpl.java

```diff
@@ -134,5 +135,22 @@ public Metadata findOneOldestByChangeDate() {
 
         return results;
     }
+    
+    /**
+     * @see org.fao.geonet.repository.MetadataRepositoryCustom#findAllSimple(org.springframework.data.jpa.domain.Specification)
+     * @param spec
+     * @return
+     */
+    @Override
+    public List<SimpleMetadata> findAllSimple(String id) {
+       Query query =  _entityManager.createQuery(
+               "select new org.fao.geonet.repository.SimpleMetadata("
+               + "id, uuid, dataInfo.changeDate, dataInfo.type_JPAWorkaround) "
+                + "from Metadata where harvestInfo.uuid = '" + id + "'");
```

Comment:
this should work:

```
_entityManager.createQuery(
                "select new org.fao.geonet.repository.SimpleMetadata("
                 "id, uuid, dataInfo.changeDate, dataInfo.type_JPAWorkaround) "
                  "from Metadata where harvestInfo.uuid = :id").setParameter("id", id);

```

Obviously untested, and probably need to be reported upstream